networkx/algorithms/centrality/tests/test_betweenness_centrality.py

```diff
@@ -563,6 +563,26 @@ def test_K5(self):
         for n in sorted(G.edges()):
             assert b[n] == pytest.approx(b_answer[n], abs=1e-7)
 
+    def test_samples_K5(self):
+        """Edge betweenness centrality: K5"""
+        G = nx.complete_graph(5)
+        b_answer = nx.edge_betweenness_centrality(G, weight=None, normalized=False)
+        b_sample = nx.edge_betweenness_centrality(
+            G, weight=None, k=G.number_of_nodes(), normalized=False
+        )
+        for n in sorted(G.edges()):
+            assert b_sample[n] == pytest.approx(b_answer[n], abs=1e-7)
+
+    def test_samples_normalized_K5(self):
+        """Edge betweenness centrality: K5"""
+        G = nx.complete_graph(5)
+        b_answer = nx.edge_betweenness_centrality(G, weight=None, normalized=True)
+        b_sample = nx.edge_betweenness_centrality(
+            G, weight=None, k=G.number_of_nodes(), normalized=True
```

Comment:
Thanks for the PR @jnke2016!

Could you add a test which tests the change by providing `k` argument? In this case we can't see any change to the result. Something like `k=0` which should raise a `ZeroDivisionError` through `_rescale_e` could be a good test, as previously it just executed the code and gave a centrality score of 0 to all edges.

[Maybe we should catch the  `ZeroDivisionError` error here @dschult?]

Reply:
To be clear, I think it would be helpful to have a test value of k that changes the output.  And the zero division error would certainly do that. So, let's include that one more test.